server/config.coffee

```diff
@@ -34,7 +34,7 @@ config =
     ]
 
     plugins: [
-        MODEL_MODULE
+        'cozydb'
```

Comment:
the pouchdb compatibility should be provided by cozydb (we discussed it in another PR, it still needs to be tested)